rox_bringup/launch/bringup_sim_launch.py

```diff
@@ -86,8 +91,29 @@ def execution_stage(context: LaunchContext, frame_type, rox_type, arm_type, use_
         name='parameter_bridge',
         output='screen',
         parameters=[{'config_file': bridge_config_file}])
+    
+    joint_state_broadcaster_spawner = Node(
+        package="controller_manager",
+        executable="spawner",
+        arguments=["joint_state_broadcaster", "-c", "/controller_manager"],
+    )
 
-    return [start_robot_state_publisher_cmd, ignition, gz_bridge, teleop, spawn_robot]
+    initial_joint_controller_spawner_stopped = Node(
+        package="controller_manager",
+        executable="spawner",
+        arguments=["joint_trajectory_controller", "-c", "/controller_manager"],
+    )
+
+    launch_actions.append(start_robot_state_publisher_cmd)
+    if arm_type != '':
+        launch_actions.append(joint_state_broadcaster_spawner)
+        launch_actions.append(initial_joint_controller_spawner_stopped)
+    launch_actions.append(ignition)
+    launch_actions.append(gz_bridge)
+    launch_actions.append(teleop)
+    launch_actions.append(spawn_robot)
```

Comment:
Only needed to be added to launch actions when ur arm is selected.

Reply:
launch_actions.append(start_robot_state_publisher_cmd)
    if arm_type != '':
        launch_actions.append(joint_state_broadcaster_spawner)
        launch_actions.append(initial_joint_controller_spawner_stopped)
        launch_actions.append(gz)
        launch_actions.append(gz_bridge)
        launch_actions.append(teleop)
        launch_actions.append(spawn_robot)
        
        like this anna? so without arm only robot state pub shoudl be run?

Reply:
Yes, something like this. People should be just able to use ROX standalone without arm on the top

Reply:
I have fixed it Pradheep, there was a typo in that condition. Now it should work fine